Sources/Misc/Deprecations.swift

```diff
@@ -389,3 +389,28 @@ extension CustomerInfo {
     }
 
 }
+
+public extension Configuration.Builder {
+    ///
+    /// Set `usesStoreKit2IfAvailable`. If `true`, the SDK will use StoreKit 2 APIs internally. If disabled, it will use StoreKit 1 APIs instead.
+    /// - Parameter usesStoreKit2IfAvailable: enable StoreKit 2 on devices that support it.
+    /// Defaults to  `false`.
+    /// - Important: This configuration flag has been deprecated, and will be replaced by automatic remote configuration in the future.
```

Comment:
Can you explain what you mean by "automatic remote configuration"?

Reply:
Hmm well, this might be going too ahead with this comment, but I believe the intention is to support changing this dynamically from the backend. So we could decide, according to the status returned by diagnostics or other sources, change whether apps use SK1 or SK2.

Reply:
Oooh that makes sense